packages/language-server/server/server.js

```diff
@@ -73,181 +37,27 @@ connection.onHover(meta => {
       ? undefined
       : {
           contents: [
-            { language: "typescript", value: getTypeName(varInfo.type || varInfo) }
-          ]
+            {
+              language: "typescript",
+              value: getTypeName(varInfo.type || varInfo),
+            },
+          ],
         };
   }
 });
 
-let timeout;
-documents.onDidChangeContent(change => {
-  clearTimeout(timeout);
-  timeout = setTimeout(() => validateTextDocument(change.document), 200);
```

Comment:
Oh, `setTimeout` was used for preventing every time re-analyzation at every keyboard button pressing, so, can you add into refactored code the snippet?